reinforcement_learning/bindings/cs/rl.net/rl.net.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>$(SolutionDir)$(Platform)\$(Configuration)</OutputPath>
+    <Platforms>AnyCPU;x64</Platforms>
```

Comment:
>AnyCPU [](start = 15, length = 6)

is AnyCPU supported?

Reply:
No, but nuget restore did not seem to do the right thing here unless I kept AnyCPU. Will try deleting again.